RELEASE-NOTES.md

```diff
@@ -9,669 +12,344 @@
-- Support specifying non-default ACL at bucket creation time
+## Configuration Mapping Table between 1.5 and 2.0
 
-# Riak CS 0.1.2 Release Notes
+Some important configuration changes happened between 1.5 and 2.0, Not
+only translating items 1:1.
 
-## Bugs Fixed
+Value following with `=` is its default value.
 
-- Return 403 instead of 503 for invalid anonymous or signed requests.
-- Properly clean up processes and connections on object requests.
+The file name has changed from `app.config` and `vm.args` to only
+`riak_cs.conf`, whose path haven't changed.
 
-# Riak CS 0.1.1 Release Notes
+### Riak CS
 
-## Bugs Fixed
+#### Items effective in the config file in 2.0 by default
 
-- HEAD requests always result in a `403 Forbidden`.
-- `s3cmd info` on a bucket object results in an error due to missing
-  ACL document.
-- Incorrect atom specified in `riak_moss_wm_utils:parse_auth_header`.
-- Bad match condition used in `riak_moss_acl:has_permission/2`.
+Note: `storage.stats.schedule.$time` does not have any default
+value but an example is added.
 
-# Riak CS 0.1.0 Release Notes
+`riak_cs` section in app.config
 
-## Bugs Fixed
+|      1.5                           |        2.0                            |
+|:-----------------------------------|:--------------------------------------|
+|`{cs_ip, "127.0.0.1"}`              |`listener = 127.0.0.1:8080`            |
+|`{cs_port, 8080}`                   |                                       |
+|`{riak_ip, "127.0.0.1"}`            |`riak_host = 127.0.0.1:8087`           |
+|`{riak_pb_port, 8087}`              |                                       |
+|`{stanchion_ip, "127.0.0.1"}`       |`stanchion_host = 127.0.0.=:8085`      |
+|`{stanchion_port, 8085 }`           |                                       |
+|`{stanchion_ssl, false }`           |`stanchion_ssl = off`                  |
+|`{anonymous_user_creation, false}`  |`anonymous_user_creation = off`        |
+|`{admin_key, "admin-key"}`          |`admin.key = admin-key`                |
+|`{admin_secret, "admin-secret"}`    |`admin.secret = admin-secret`          |
+|`{cs_root_host, "s3.amazonaws.com"}`|`root_host = s3.amazonaws.com`         |
+|`{connection_pools,[`               |                                       |
+|` {request_pool, {128, 0} },`       |`pool.request.size = 128`              |
+|                                    |`pool.request.overflow = 0`            |
+|` {bucket_list_pool, {5, 0} }`      |`pool.list.size = 5`                   |
+|                                    |`pool.list.overflow = 0`               |
+|`{trust_x_forwarded_for, false}`    |`trust_x_forwarded_for = off` *          |
+|`{leeway_seconds, 86400}`           |`gc.leeway_seconds = 24h`              |
+|`{gc_interval, 900}`                |`gc.interval = 15m`                    |
+|`{gc_retry_interval, 21600}`        |`gc.retry_interval = 6h`               |
+|`{access_log_flush_factor, 1}`      |`access.stats.log.flush_factor = 1`    |
```

Comment:
Discussion: I think `access.stats.flush_factor` (without `log.`) is simpler and sufficient.
Also access log and access stats are confusing, so I prefer to not using `log` here in the context of access stats.